### DISCUSSION TITLE
Static HTML Export not working with any hooks

### DISCUSSION THREAD

**Discussion by jonrrivera** [vercel/next.js#40527] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

yarn install & yarn build

### Which example does this report relate to?

https://github.com/vercel/next.js/tree/canary/examples/with-static-export

### What browser are you using? (if relevant)

chrome

### How are you deploying your application? (if relevant)

static [local]

### Describe the Bug

Hi, 

I've been trying to run a Static HTML Export on my app. I've found that every time I export following these [instructions](https://nextjs.org/docs/advanced-features/static-html-export),  I get these console errors when opening the exported html file in the out folder:

<img width="1458" alt="Screen Shot 2022-09-13 at 1 03 36 PM" src="https://user-images.githubusercontent.com/18686388/189998279-ec7e87ce-dff1-4b4d-a477-0c53d6a36157.png">

Should I expect react hooks (useState, useEffect) to work when running my statically exported app? I can't find that limitation noted on the documentation. 

Any help here would be great, thank you!

### Expected Behavior

running `yarn build` would generate a static site that works with react hooks. 

### To Reproduce

https://github.com/vercel/next.js/tree/canary/examples/with-static-export

download, run and `yarn export` the example site. Open the `index.html` file in the out folder, and see the missing core.js files in the console.

**Comment by ijjk** [comment]
Hi, you need to still serve the assets using something like https://npmjs.com/package/serve as the references aren't relative so won't load when opened directly from the filesystem.

**Reply by jonrrivera** [reply]
thank you! going to try this out. 

Figured it would be nice to have this info in the [documentation](https://nextjs.org/docs/advanced-features/static-html-export).

**Reply by jonrrivera** [reply]
<img width="1492" alt="Screen Shot 2022-09-13 at 2 42 07 PM" src="https://user-images.githubusercontent.com/18686388/190014563-d616ee25-487d-46f7-bc00-4443dcfa7c9f.png">

still getting the same issue when serving using serve :/
